sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -318,7 +331,7 @@ def __init__(
     # of this map should last as long as the corresponding entry in _tag_map.
     self._proxy_map: Dict[str, multi_process_shared.MultiProcessShared] = {}
 
-  def load(self, key: str) -> str:
+  def load(self, key: str) -> _ModelLoadStats:
```

Comment:
Should we change the doc for Return to `_ModelLoadStats with tag, byte size and latency to load the model`

Reply:
Yes, good catch - updated!